docs/external-oasis-repos.md

```diff
@@ -30,31 +30,43 @@ Requests to Host a Repository on OASIS
-    When PR is approved, check on the `oasis.opensciencegrid.org` host whether the new repository was successfuly signed.
-    There should be sign of it in the log file `/var/log/oasis/generate_whitelists.log`:
+    When the PR is approved, check on the `oasis.opensciencegrid.org` host whether the new repository was successfuly signed.
+    There should be message about it in the log file `/var/log/oasis/generate_whitelists.log`:
 
     ```
     Tue Sep 25 17:34:02 2018 Running add_osg_repository http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/dune.osgstorage.org
 dune.osgstorage.org: Signing 7 day whitelist with masterkeycard... done
     ```
 
-1.  If the respository ends in a new domain name `domain.name` that has not been distributed before, then place a copy of the
-    `domain.name.pub` public key into `/srv/etc/keys` on both `oasis-replica` and
-    `oasis-replica-itb`. If you do not have that key, then ask the repository service representative how to obtain it. In
-    order to support CVMFS client versions 2.2.X, also make a symbolic link of `domain.name.conf`
-    `/cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d` pointing to `default.conf`. This symbolic link has to be
-    created on the `oasis-itb` machine's copy of the `config-osg.opensciencegrid.org` repository and then copied to
-    production with the `copy_config_osg` command on the oasis machine.
+1.  If the respository ends in a new domain name that has not been distributed before, 
+    a new domain key will be needed on oasis-replica which should get automatically downloaded from github.
+    There should be a message about downloading it in the log file `/var/log/cvmfs/generate_replicas.log`.
+    The key has to first exist in the master branch of the [config-repo github repository](https://github.com/cvmfs-contrib/config-repo).
+    After the key is downloaded the repository should also be automatically added, with messages in the same log file.
+
+    After the repository is successfully on oasis-replica, in addition you need to update the
+    OSG configuration repository.  Make changes in a workspace cloned from the
+    [config-repo github repository](https://github.com/cvmfs-contrib/config-repo)
+    and use the osg branch (or a branch made from it) in a personal account on oasis-itb.
+    Add a domain configuration in `etc/cvmfs/domain.d` that's a lot like one of the other imported domains, for example `egi.eu.conf`.
+    The server urls might be slightly different; use the URLs of the stratum 1s where it is already hosted if there are any,
+    and you can add at least the FNAL and BNL stratum 1s.
+    Copy key(s) for the domain into `etc/cvmfs/keys` from the master branch, either a single .pub file or a directory, whichever the master branch has.
+    Test all these changes out on the `config-osg.opensciencegrid.org` repository on oasis-itb
+    using the `copy_config_osg` command and configure a test client to read from oasis-itb instead of oasis to test it.
```

Comment:
should we use instead short hostname full hostname everywhere (DNS alias)?

Reply:
I think the short hostnames are clear enough.

Reply:
I changed my mind.  In this case since it's talking about client configuration the full host name makes sense.  I'll change it.